templates/ftp_listener/ballerina.conf

```diff
@@ -0,0 +1,8 @@
+FTP_HOST = "192.168.112.8"
```

Comment:
Shall we remove the ip and keep it blank? Also, let's keep some default values for the other properties also.

Reply:
Updated.